src/common/pages/submit.tsx

```diff
@@ -216,7 +216,6 @@ class SubmitPage extends BaseComponent<Props, State> {
 
     this.detectDraft().then();
 
-    this.checkDraft();
```

Comment:
why removing this check?

Reply:
@MTSaeed also, what happens if user doesn't have any drafts stored on backend?

Reply:
> why removing this check?

Actually I did the working of this function in the loadLocalDraft function.That's why i think there is no more need to create extra function.

Reply:
> @MTSaeed also, what happens if user doesn't have any drafts stored on backend?

It has the current functionality of the Drafts popup. If there are no drafts, the button is still displayed and the drafts pop up opens. It shows the default message for no drafts.